client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -49,11 +50,45 @@
 
     private final int maxOutstandingCheckpointRequest;
 
+    private final StreamDataRetention retentionConfig;
+
+    /**
+     * If a Stream's Retention Policy {@link RetentionPolicy} is set to 'CONSUMPTION' based,
```

Comment:
"Consumption based" is not clear. Please link to the exact setting explicitly.

Reply:
That retention type was added by another PR and not available till this morning. I have added a reference to it now.